codespell_lib/data/dictionary.txt

```diff
@@ -31387,6 +31449,7 @@ sophmore->sophomore
 sorce->source, force,
 sorceror->sorcerer
 sord->sword, sore, sored, sawed, soared,
+soring->sorting
```

Comment:
Just discovered [soring](https://en.wikipedia.org/wiki/Soring), but I doubt the word has been used on purpose in source code. Could be moved to `rare`, or not...